steam_wrapper/steam_wrapper.py

```diff
@@ -125,7 +126,18 @@ def check_allowed_to_run(current_info):
         shutil.rmtree(steam_home)
 
 
-def migrate_config():
+def make_xdg_ignores(current_info, xdg_name, path):
+    ignores = []
+    filesystems = current_info["filesystems"]
+    for filesystem in filesystems:
+        filesystem_path = filesystem.split(":")[0]
+        if filesystem_path.startswith(xdg_name):
+            ignores.append(re.sub(f"^{xdg_name}", path,
```

Comment:
Why use regex here? Maybe just use `os.path.split()` here to turn string into sequence of path components?